src/main/kotlin/xoxo/xoxo.kt

```diff
@@ -60,6 +64,8 @@ class XmlText internal constructor(private val text: Text) : XmlNode {
     val content: String
         get() = text.textContent
 
+    override fun asDomNode(): Text = element
```

Comment:
Woops, I merged this a bit too fast, this should have been ` = text`. Fixed in https://github.com/martinbonnin/xoxo/pull/16/files#diff-e46bd4e6885bb3012177d2423c2f36670ee5a1ccfeff883afdc713adc39c318dR67